File: matiasmasso/Mat.NET/DTO/DTOImportPrevisio.cs
using System;
using System.Collections.Generic;

namespace DTO
{
    public class DTOImportPrevisio : DTOBaseGuid
    {
        public DTOImportacio Importacio { get; set; }
        public string NumComandaProveidor { get; set; }
        public int Lin { get; set; }
        public string SkuRef { get; set; }
        public DTOProductSku Sku { get; set; }
        public string SkuNom { get; set; }
        public int Qty { get; set; }
        public DTOPurchaseOrderItem PurchaseOrderItem { get; set; }
        public DTOInvoiceReceived.Item InvoiceReceivedItem { get; set; }

        public List<ValidationErrors> errors { get; set; }

        public enum ValidationErrors
        {
            skuNotFound,
            orderNotFound
        }

        public enum ValidationSolutions
        {
            selectSku,
            selectOrder
        }

        public DTOImportPrevisio() : base()
        {
            errors = new List<ValidationErrors>();
        }

        public DTOImportPrevisio(Guid oGuid) : base(oGuid)
        {
            errors = new List<ValidationErrors>();
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOImportPrevisio.vb
Public Class DTOImportPrevisio
    Inherits DTOBaseGuid
    Property Importacio As DTOImportacio
    Property NumComandaProveidor As String
    Property Lin As Integer
    Property SkuRef As String
    Property Sku As DTOProductSku
    Property SkuNom As String
    Property Qty As Integer
    Property PurchaseOrderItem As DTOPurchaseOrderItem

    Property Errors As List(Of ValidationErrors)

    Public Enum ValidationErrors
        SkuNotFound
        OrderNotFound
    End Enum

    Public Enum ValidationSolutions
        SelectSku
        SelectOrder
    End Enum

    Public Sub New()
        MyBase.New()
        _Errors = New List(Of ValidationErrors)
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
        _Errors = New List(Of ValidationErrors)
    End Sub

End Class
